extensions/docker/package.json

```diff
@@ -33,6 +33,16 @@
       "mode": "view"
     }
   ],
+  "preferences": [
+    {
+      "name": "socketPath",
+      "type": "textfield",
+      "required": false,
+      "title": "Socket path",
+      "description": "Path to Docker socket",
+      "placeholder": "e.g. /var/run/docker.sock"
```

Comment:
Might be simply a matter of taste but perhaps simply `/var/run/docker.sock` would be cleaner.